pkg/tests/cross-tests/pu_test.go

```diff
@@ -0,0 +1,103 @@
+package crosstests
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/pulumi/providertest/providers"
+	"github.com/pulumi/providertest/pulumitest"
+	"github.com/pulumi/providertest/pulumitest/opttest"
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge/tokens"
+	shimv2 "github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfshim/sdk-v2"
+	"github.com/stretchr/testify/require"
+)
+
+func pulTest(t *testing.T, providerName string, resMap map[string]*schema.Resource, program string) *pulumitest.PulumiTest {
```

Comment:
The file is called `pu_test.go`, this function is `pulTest`. I don't think we need to abbreviate here, but we certainly shouldn't abbreviate inconsistently.

```suggestion
func pulumiTest(t *testing.T, providerName string, resMap map[string]*schema.Resource, program string) *pulumitest.PulumiTest {
```